### ISSUE TITLE
Agenda for July 25th, 2024

### ISSUE THREAD

**Issue by nairnandu** [web-platform-tests/interop#679]
Here is the proposed agenda for July 25th:
- Review and finalize #657 
- Review proposal templates
- Test change proposals

**Comment by nairnandu** [comment]
Attendees: @nairnandu, @foolip, @meyerweb, @bkardell, @jgraham, @nsull, @nt1m, @dandclark

Notes



*   Review and finalize [2025 proposal selection process #657](https://github.com/web-platform-tests/interop/pull/657)
    *   nairnandu: call to merge the process PR 
    *   dandclark: need some more time for internal reviews
    *   nsull: +1 would like to have a final round of review internally
    *   Merging of PR deferred to Aug 8th.
*   Review proposal [templates](https://github.com/web-platform-tests/interop/pull/680)
    *   foolip: updated the proposal template dates
    *   jgraham: will draft a PR to include the outcomes from the signals discussion
*   [Test change proposals](https://github.com/web-platform-tests/interop/issues?q=is%3Aissue+is%3Aopen+label%3Atest-change-proposal)
    *    #389 - waiting on WPT PR review
    *   #634 - follow-up
    *   #668 - resolved
    *   #670 - spec issue still open. Re-visit
    *   #673 - follow-up
    *   #677 - follow-up on the actual issue
*   URL focus area
    *   nsull: we expect some compat problems and we would like to make a coordinated effort to fix the last remaining failures
    *   jgraham: file an issue with the impact so that others can review and chime in
    *   foolip: +1 on the compat risk and for the discussion
